### PR TITLE
Sb 14223 merge user-cert api changes

### PR DESCRIPTION
**Pull request by Hari-stackroute**
created method for mergeUserCertificate for merging the user-certificates

### CODE REVIEW

File: actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java [hunk 1]
```diff
@@ -17,151 +13,166 @@
...
+        String mergerId = (String) request.get(JsonKey.TO_ACCOUNT_ID);
+        Response response = cassandraOperation.getRecordsByProperty(certDbInfo.getKeySpace(),certDbInfo.getTableName(),JsonKey.USER_ID,mergeeId);
+        Map<String, Object> record = response.getResult();
+        if (null != record && null != record.get(JsonKey.RESPONSE)) {
+            List responseList = (List) record.get(JsonKey.RESPONSE);
+            if (!responseList.isEmpty()) {
+                responseList.forEach(responseMap -> {
+                    Map<String, Object> responseDetails = (Map<String, Object>) responseMap;
+                    responseDetails.put(JsonKey.USER_ID,mergerId);
+                    cassandraOperation.updateRecord(certDbInfo.getKeySpace(),certDbInfo.getTableName(),responseDetails);
```
Comment (manzarul) [inline actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java]: instead of singlr update we can used bulk update method.

Comment (manzarul) [inline actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java]: Add some logs for success as well.

Comment (Hari-stackroute) [reply actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java]: The provided method for bulk upload is not working as expected, so proceeding with this.

Comment (Hari-stackroute) [reply actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java]: Done

Comment (manzarul) [reply actors/user/src/main/java/org/sunbird/user/actors/CertificateActor.java]: Ok

**Pull request merged**